### DISCUSSION TITLE
2+Players Horsengel Roulette

### DISCUSSION THREAD

**Discussion by Horsengel** [helmasaur/horsengel-roulette#48] [Ideas]
The project relative to a Horsengel roulette played as a Tournament was suggested in the following topic : https://github.com/helmasaur/horsengel-roulette/discussions/46
I had two main ideas for this : 
- Creating brackets in which players can play duels and go (or not) to semifinals and finals. If the number of people is odd, one of them (randomly choosed) would directly go on the second round.
- Doing the same as a classical 2-players Horsengel Roulette : each player pulls the trigger one at a time, the shot would result on a kick 1/6th of the time (Another idea would be to choose the number of chambers in the "gun", therefore the probability of a losing turn). As people looses, there would be only one person left, this person would win the tournament.
Another idea would be to increase the number of chambers to the number of participants. For instance : If 7 people play, there would be a 1/7 chance of being eliminated. On the second turn, 1/6 and so on.

**Comment by helmasaur** [comment]
OK, so as I thought, it's not the same idea than what I thought for #4 (I know that the issue doesn't describe much. I thought about multiplay as the same gun passed from one to the other and either there would be one loser and all the others winners, either add more bullets and then, there would only be one winner. Your idea would be to give a different gun at each rounds, right?

If I understood well, it would be a command like this; `!hr [@player1, @player2, …, @playerX]`? And applied to an example:
1. `!hr [@player1, @player2, @player3, @player4]`
2. player1 vs player2 (winner: player1)
3. player3 vs player 4 (winner: player4)
4. player1 vs player4

For the amount of bullets and the chamber size, the code is normally ready for that (variables are already there).
